src/map-from.ts

```diff
@@ -62,4 +62,20 @@ export const mapFrom = {
   undefined(): undefined {
     return undefined;
   },
+
+  /**
+   * Maps the specified properties to themselves as is
+   *
+   * ```ts
+   * const objectMapperSchema = {
+   *   ...mapFrom.pick('a', 'b', 'c', 'd'),
+   * }
+   * ```
+   */
+  pick<TKey extends PropertyKey>(...keys: TKey[]) {
```

Comment:
🐛 `mapFrom.pick()` can be called with zero arguments. _(There are "stupid type tricks" you can do to ensure at least one arg is provided)_
🙏 Can we change `keys` to be a single array argument, rather than variadiac args? _(This would require similar but different type tricks, to require at least one array element)_

Also:
⛏️ Add the return type to the function signature.
⛏️ Keep the `mapFrom` items in alphabetical order (`mapFrom.pick` should be before `mapFrom.undefined`)

Reply:
☑️ Added explicit return type 👌 
☑️ Re-ordered methods alphabetically 👌